### ISSUE TITLE
Project Sorting by Oldest First is Problematic

### ISSUE THREAD

**Issue by SColchester** [hotosm/tasking-manager#5138]
**Problem**
Projects are first sorted by Priority; this is perfect and makes sense. The most urgent projects are at the front. However, within this, projects are then sorted to show the oldest projects first. This makes sense to a certain extent; older projects need more attention to get completed. However, this results in the first page of the Tasking Manager being **dominated by old projects**. Right now, five of the fourteen projects on the front page are more than two years old, seven are more than a year old and **only two** (the urgent ones) are recent projects. See below:

![2022-05-19 18_36_01-Explore projects - HOT Tasking Manager — Mozilla Firefox- Edit](https://user-images.githubusercontent.com/43822967/169353426-23b7a437-2525-45ae-91a7-6cb5185967f9.png)

Among other issues with old projects, they often outlive their original purpose - this means that we are constantly at risk of directing lots of mappers to projects where the data needs have moved on. Volunteer time could be spent better on more contemporary projects, where the data needs are still present. 

**Solution**
Sort projects by priority, but for the second ordering opt for something else than oldest first. This could be most recent projects first. Or it could be oldest projects within a certain timeframe (e.g., last 3 months). The solution to this is probably best discussed in a TM collective meeting - it matters too much to be decided single-handedly. 

**Alternatives**
If project quality could be measured automatically this might be a great thing to sort by. E.g. projects that have the most complete instructions and descriptions

**Additional context**
As far as I am aware this sort by oldest first occurred with the most recent Tasking Manager, inspiration might come from looking back to see how projects were sorted in older Tasking Manager versions.

**Comment by SColchester** [comment]
@russdeffner interested to hear your perspective on this 😃

**Comment by russdeffner** [comment]
I believe it is the lesser of two evils; when it sorted the other way (newest listed first) then it was unfair to the folks who were getting traction and about to get their project complete. In other words, you could be quite cruel and pop in a new project when the project at the number 1 spot was only hours away from getting complete and then suddenly that project drops off the top and then further and further down 'the hole' and never gets complete. Places oldest first gives incentive that wouldn't exist otherwise - one being, to try and get projects completely finished so we can archive them and two - for project managers to sequence their projects properly (i.e. if I have 3 projects for my campaign and create them all at once, I can just put one at Urgent/High and let it get complete, then rotate in the next, and due to project numbering it will take the spot of the one I just archived and so on. Doing it the other way the only incentive is for me to put all my projects up at once.)

In general - I vote for keeping the order this way (oldest on top / bottom load new projects).

**Comment by RAytoun** [comment]
The sorting criteria is complex and cannot be dealt with by simply adding a single default. As I have shown the two Durban flood projects disappeared off down to page 6 and the Afghanistan Earthquake disappeared off down to page 3.

One of the problems is some of the campaigns are churning out projects at an alarming rate as with the Open Cities Guatemala which has 22 pages of projects 

https://tasks.hotosm.org/explore?campaign=OpenCities%20LAC&location=Guatemala&page=22  

While many of them are mapped they are not validated and the beginner mappers never stay long enough to build up the expertise to become validators. This is about mapping a whole country, not mapping for humanitarian purposes.

**Comment by zlavergne** [comment]
I agree @RAytoun. My suggestion above is exactly that, to not use a single metric for the default sorting method. But a default sorting method is necessary, otherwise we wouldn't be able to show anything on the projects page for new users. 

I want to make sure I understand the problem you're describing. Can you elaborate on the issue with Open Cities Guatemala's projects taking up 22 pages? Are you saying that these projects should be sorted lower down because they are not humanitarian efforts? In other words, humanitarian projects should always be higher on the list. (That's something we could do that would be easy from an implementation standpoint but will need consideration from an adoption standpoint)

**Comment by aawiseman** [comment]
i agree, it would be good to improve the sorting

**Comment by SColchester** [comment]
The latest version of Tasking Manager (TM5) seems to take a new approach with project sorting - still showing urgent priority first but then not necessarily showing the oldest projects first anymore. It would be good to know how the new sorting method works.  (FYI @ramyaragupathy @russdeffner @RAytoun @zlavergne )
